poradnia/stats/mixins.py

```diff
@@ -0,0 +1,92 @@
+from django.core.exceptions import ImproperlyConfigured
+from guardian.shortcuts import assign_perm
+
+from users.factories import UserFactory
+
+
+class PermissionStatusMixin(object):
+    """Mixin to verify object permission status codes for different users
+    Require user with username='john' and password='pass'
+    Attributes:
+        permission (str): Permission name or "superuser"
+        status_anonymous (int): Status code for anonymouser
+        status_has_permission (int): Status code for user with permission
+        status_no_permission (403): Status code for user without permission
+        url (str): url to test
+    NOTE: based on https://github.com/watchdogpolska/feder/blob/master/feder/main/mixins.py#L113
```

Comment:
To być powinien odnośnik nie do brancha, a do commita, bo branch może ulec zmianie. Poprawię.